RELEASE.md

```diff
@@ -0,0 +1,253 @@
+# Release Process
+
+## Introduction
+
+The process for managing enhancements (OKEPs), issues, and pull requests through an OVN-Kubernetes release is complex. This document describes the process in detail.
+
+## How often are releases?
+
+We release approximately once every 3 calendar months.
+
+## How do we decide on the release date?
+
+The release date is decided during the [community meeting] at the beginning of the release cycle.
+It will be approximately 3 calendar months from the previous release. The release date will not land on a Friday, holiday, or weekend.
+
+## Who is responsible for the release?
+
+The release coordinator is elected at the [community meeting] at the beginning of the release cycle.
+The release coordinator may be any volunteer from the community - a call for volunteers will be issued one week before the next community meeting via the community Slack and mailing list.
+
+### Release Coordinator Responsibilities
+
+- The release coordinator is the person who is ultimately responsible for ensuring that the release is completed on time, and with the desired payload.
+- The release coordinator follows the release process outlined in [#what-is-the-release-process].
+- The release coordinator ensures that all community members are following the correct process.
+- While not a maintainer, the release coordinator is an important role in the community and should be afforded the respect and trust of the maintainers.
+- The release coordinator is empowered to make decisions about the release that are in the best interest of the community.
+  - For example, the release coordinator may choose to delay the release if the payload is not ready or if the quality of the release is not high enough.
+- The release coordinator is responsible for ensuring that progress is being made on the release and regularly updates the community on the progress of the release during the [community meeting].
+- If decisions are required that are outside the scope of the release coordinator's responsibilities, the release coordinator will seek the advice of the maintainers.
+- If the release coordinator is unable to fulfill their responsibilities, they may step down and a new release coordinator will be elected.
```

Comment:
```suggestion
- If the release coordinator is unable to fulfill their responsibilities, they must step down and a new release coordinator will be elected.
```

I suggest stronger wording. I mean, after all, the release coordinator person is *unable* to fulfill their responsibilities. IMHO, they *must* step down.

Thinking a bit more about this, I think there also needs to be a process for the release coordinator to be "released" from their position by the community (maintainers, probably) - e.g. person is on long term sick leave, and the process must continue (just an example).